### PR TITLE
Corrected pkgconfig files

### PR DESCRIPTION
**Pull request by pierrebizz**
Fix for issue https://github.com/DOCGroup/ACE_TAO/issues/1750

### CODE REVIEW

File: TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in [hunk 1]
```diff
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: TAO_CosProperty_Serv
 Description: TAO CosProperty Service Library
-Requires: TAO_PortableServer_Skel
+Requires: TAO_PortableServer
```
Comment (jwillemsen) [inline TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in:8]:
``` suggestion
Requires: TAO_CosProperty_Skel
```

File: TAO/tao/Monitor/TAO_Monitor.pc.in [hunk 2]
```diff
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: TAO_Monitor
 Description: TAO Monitor Library
-Requires: TAO_PortableServer, TAO_ETCL_Parser, ACE_MonitorControl
+Requires: TAO_PortableServer, ACE_ETCL_Parser
```
Comment (jwillemsen) [inline TAO/tao/Monitor/TAO_Monitor.pc.in:8]: ACE_MonitorControl should be there, looks a pc.in is missing in https://github.com/DOCGroup/ACE_TAO/tree/master/ACE/ace/Monitor_Control

**Review by jwillemsen — CHANGES_REQUESTED** [review]

**Comment by mitza-oci** [comment]
I have some related changes on this branch: https://github.com/mitza-oci/ACE_TAO/tree/gha-install-pc
If it works in GitHub Actions I'll create a PR which could replace this one.
That should at least make the pkg-config files consistent (`pkg-config --exists` works), if not 100% accurate.

**Comment by mitza-oci** [comment]
See #1752

**Pull request closed without merging**